internal/builder/builder.go

```diff
@@ -1175,11 +1132,102 @@ func (b *Builder) whiteoutLayer(tmpDir string, i int, bpInfo dist.ModuleInfo) (s
 	return fh.Name(), nil
 }
 
-func sortKeys(collection map[string]toAdd) []string {
+func sortKeys(collection map[string]moduleWithDiffID) []string {
 	keys := make([]string, 0, len(collection))
 	for k := range collection {
 		keys = append(keys, k)
 	}
 	sort.Strings(keys)
 	return keys
 }
+
+// splitBuildModules takes the given Build Modules and concurrently read their internal tar files. In
+// case a flattened module is found, it will split each flattened buildpack into an individual module and skip all the empty ones
+// returns an explodedBuildModule array with the Build Module information but also the diffId and the tar file on disk
+func splitBuildModules(kind, tmpDir string, additionalModules []buildpack.BuildModule, logger logging.Logger) ([]moduleWithDiffID, []error) {
+	lids := make([]chan modInfo, len(additionalModules))
```

Comment:
naming nit  -- there might be a convention i don't know, but  the name "lids" sounds either like it's a collection of the tops to jars or containers (lid) or like it's an ID for an L-word (L. IDs),  but neither of those suggests "lists of channels" to me? 

 e.g. would `modInfoChans` or even  be an ok compromise on the spectrum of short names vs. informative names?

Reply:
Sorry, this one was just a copy from a previous method.